packages/dashmate/src/listr/prompts/validators/validateTenderdashNodeKey.js

```diff
@@ -0,0 +1,46 @@
+const crypto = require('node:crypto');
+
+/**
+ * @param {string} value
+ * @returns {boolean|string}
+ */
+function validateTenderdashNodeKey(value) {
+  if (value.length < 1) {
+    return 'should not be empty';
+  }
+
+  const nodeKey = Buffer.from(value, 'base64');
+
+  if (nodeKey.length !== 64) {
+    return 'invalid format';
+  }
+
+  const privateKey = nodeKey.slice(0, 32);
+  const privateKeyDer = Buffer.concat([
+    Buffer.from('302e020100300506032b657004220420', 'hex'), // Static value
+    privateKey,
+  ]);
+
+  const derivedPublicKeyDer = crypto.createPublicKey({
+    format: 'der',
+    type: 'pkcs8',
+    key: privateKeyDer,
+  }).export({
+    format: 'der',
+    type: 'spki',
+  });
+
+  const publicKey = nodeKey.slice(32);
+  const publicKeyDer = Buffer.concat([
+    Buffer.from('302a300506032b6570032100', 'hex'), // Static value
```

Comment:
Why are we contcatenating this static value with the public key? Is it sort of a network id?

Reply:
It's some metadata for DER key format. Always the same for all keys.

Reply:
https://keygen.sh/blog/how-to-use-hexadecimal-ed25519-keys-in-node/ let's resolve this one?

Reply:
Yeah, I got it there and tested to make sure it works with other keys.